WordPress/src/main/java/org/wordpress/android/util/WPUrlUtils.java

```diff
@@ -0,0 +1,45 @@
+package org.wordpress.android.util;
+
+import java.net.URI;
+import java.net.URL;
+
+public class WPUrlUtils {
```

Comment:
I don't think `synchronized` is needed here, since we're not modifying shared variables from multiple threads.